zebra-consensus/src/primitives/groth16/params.rs

```diff
@@ -15,19 +15,19 @@ use crate::BoxError;
 /// But `zebrad start` downloads blocks at the same time, so we allow some extra time.
 pub const PARAMETER_DOWNLOAD_TIMEOUT: u64 = 60 * 60;
 
-/// The maximum number of times to retry download parameters.
+/// The maximum number of times Zebra tries to download the parameters.
 ///
-/// Zebra will retry to download Sprout of Sapling parameters only if they
-/// failed for whatever reason.
-pub const PARAMETER_DOWNLOAD_MAX_RETRIES: usize = 3;
+/// Zebra will retry the download only if the previous attempt fails.
+pub const PARAMETER_DOWNLOAD_MAX_ATTEMPTS: usize = 3;
```

Comment:
Optional:

Use retries and make `retries + 1` attempts.

Reply:
Done in [4208704](https://github.com/ZcashFoundation/zebra/pull/6464/commits/4208704c92c446422358baa3b89fa561473fff04).